### DISCUSSION TITLE
Ability to handle "unblockable trackers" that leverage CNAME records?

### DISCUSSION THREAD

**Discussion by WiseOneChan** [Eloston/ungoogled-chromium#1897] [Ideas]
**Is your feature request related to a problem? Please describe.**
Apparently there is a new way to track users by tricking the browser into thinking the cookie stored is a 1st party cookie (https://nakedsecurity.sophos.com/2019/11/25/ad-blocking-companies-block-unblockable-tracker/). Firefox has an API to look up the real domain behind a CNAME. Are there any plans to introduce some native ability to do something similar in this project?

**Describe the solution you'd like**
Some way of exposing the real domain behind CNAME records.

**Describe alternatives you've considered**
Using Firefox

**Additional context**

**Comment by Eloston** [comment]
I think it would be a nice feature to have in the browser. Does anyone know if Chromium or any Chromium forks are already working on this?

**Comment by emanruse** [comment]
> Does anyone know if Chromium or any Chromium forks are already working on this?

https://en.wikipedia.org/wiki/UBlock_Origin#CNAME-uncloaking_third-party_trackers_in_Firefox

https://github.com/gorhill/uMatrix/commit/9b292304d33a44465922200efa5f8b378d0f9604

@WiseOneChan Do you know of any particular cloaked CNAME we can test with?
